test/e2e/support/custom.ts

```diff
@@ -370,6 +370,14 @@ Cypress.Commands.add('reloadAndWait', () => {
   cy.injectAxe();
 });
 
+Cypress.Commands.add('waitForLoad', () => {
+  cy.get('#readyForPrint').should('exist');
+  cy.findByRole('progressbar').should('not.exist');
+  // An initialOption can cause a save to occur immediately after loading is finished, wait for this to finish as well
+  cy.waitUntilSaved();
+  cy.log('App finished loading');
```

Comment:
Nice! Doesn't this overlap with what happens in `reloadAndWait` and `clearSelectionAndWait`? We should possibly consider merging a few of these, and/or at least re-using them in each other.

Reply:
Yeah, it definitely overlaps, there was just nothing that exactly fit my needs 😅 

The datepicker test was flaky due to patch requests being called on load because of option preselection or metadata or something, so I needed to wait for loading to complete before starting to count patch requests

Reply:
Using it in `reloadAndWait` caused some strange issues in the signing-tests when reaching the receipt, it would first detect that unsaved-changes attribute was present, but then on the immediate next `cy.get(body)` the attribute was no longer there (since its not present in receipt) and it would time out waiting for it to be false. I tried to figure out why this happened, but I don't feel like spending more time on it.